### PR TITLE
Companies View

### LINKED ISSUE #1769: Bug: Page not found (404)
**Describe the bug**
/companies is not valid URL 
**To Reproduce**
1. Go to compaines 
2. See error

**Screenshots**

Screenshot 2024-02-13 230407

### PR DESCRIPTION
**Pull request by HanilJain**
### Fixed #1769 

Screenshot 2024-02-16 183949

on clicking the name will direct them to /company/domain/<int:pk>

### CODE REVIEW

**Comment by HanilJain** [comment]
@DonnieBLT could you please review ?

File: website/views.py [hunk 1]
```diff
@@ -3503,116 +3503,24 @@ def safe_redirect(request: HttpRequest):
     fallback_url = f"{request.scheme}://{request.get_host()}/"
     return redirect(fallback_url)
 
-# class CreateIssue(CronJobBase):
-#     RUN_EVERY_MINS = 1
-
```
Comment (DonnieBLT) [inline website/views.py:3508]: This code is a future feature please revert

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
One tweak

File: website/views.py (hunk 1 above)
Comment (HanilJain) [reply website/views.py:3508]: Ohh sorry didn't knew about it. Revert done 0ec0ea76c7abbcd29983cede000687c627622b26

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**